File: solucoes/_exemplo2/_exemplo2.py
# Exemplo de uso do repositório: concatenação de n textos
# teste com: "python validar.py _exemplo2.py" no terminal

import sys

inp = sys.stdin.readline
out = sys.stdout.write

n = int(inp())
texts = [inp().strip() for _ in range(n)]

out("".join(texts))

File: solucoes/_exemplo2/_exemplo2.cpp
// Exemplo de uso do repositório: concatenação de n textos
// teste com: "python validar.py _exemplo2.cpp" no terminal

#include <iostream>

using namespace std;

int main() {
    ios::sync_with_stdio(0);
    cin.tie(0);

    int n;
    cin >> n;

    string output = "";
    for (int i = 0; i < n; ++i) {
        string text;
        cin >> text;
        output += text;
    }

    cout << output << "\n";
}